packages/terra-consumer-nav/src/components/nav-toggler/NavToggler.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import OutlineChevronDown from '../../icons/OutlineChevronDown';
+import OutlineChevronUp from '../../icons/OutlineChevronUp';
+import './NavToggler.scss';
+
+const propTypes = {
+  text: PropTypes.string,
+  subItems: PropTypes.arrayOf(
+    PropTypes.shape({
+      text: PropTypes.string,
+      uri: PropTypes.string,
+      isActive: PropTypes.bool.isRequired,
+    }),
+  ),
+  handleToggle: PropTypes.func.isRequired,
+  isCollapsed: PropTypes.bool,
+  setActive: PropTypes.func,
```

Comment:
generally callbacks should be labeled `onToggleChange` or something.  But in this case what needs to know about the toggle change?